### ISSUE TITLE
Automating website

### ISSUE THREAD

**Issue by himanshu007-creator** [Canvasbird/canvasboard#332]
I would like to work on this issue.I will create a script that will enable opening website from CLI.
Please add DWOC label to it and assign me this issue.

**Comment by goliakshay357** [comment]
Hey @himanshu007-creator,
Can you please describe briefly? 

Thanks

**Comment by himanshu007-creator** [comment]
> Hey @himanshu007-creator,
> Can you please describe briefly?
> 
> Thanks

Yes sure, basically I will create a bash script, when user runs this script on CLI, website will open in browser automatically.
It will be helpful for those using ubuntu and similar operating systems.

**Comment by goliakshay357** [comment]
Hey @himanshu007-creator,
How it is different from ``` ng serve -o ``` ?

The command "-o" will open the local hosted angular website automatically  in the default browser.

**Comment by himanshu007-creator** [comment]
> Hey @himanshu007-creator,
> How it is different from `ng serve -o` ?
> 
> The command "-o" will open the local hosted angular website automatically in the default browser.

sir, my script will not open the locally hosted website, but it will open the main website.
https://canvasboard.live/home
this one

**Comment by kingavatar** [comment]
Hey @himanshu007-creator ,
Please correct me if i am wrong, so the bash script will automatically open canvasboard.live link from terminal. But how is it different from say opening using firefox command like `firefox canvasboard.live` as it also opens weblinks from terminal, are there any other features which are also being automated.

**Comment by himanshu007-creator** [comment]
> Hey @himanshu007-creator ,
> Please correct me if i am wrong, so the bash script will automatically open canvasboard.live link from terminal. But how is it different from say opening using firefox command like `firefox canvasboard.live` as it also opens weblinks from terminal, are there any other features which are also being automated.

Sir, I don't know about firefox command. If you feel my contribution will not add value to project, then you can decline my services.

**Comment by goliakshay357** [comment]
Hey @himanshu007-creator,
We always welcome new ideas but can you briefly explain the issue step-wise? Would be great if you can follow the issue  template provided. This would help us in judging whether the issue fits the improvements gap.

Thanks

**Comment by himanshu007-creator** [comment]
> Hey @himanshu007-creator,
> We always welcome new ideas but can you briefly explain the issue step-wise? Would be great if you can follow the issue template provided. This would help us in judging whether the issue fits the improvements gap.
> 
> Thanks

---
name: Feature request
about: Suggest an idea for this project
---

**Is your feature request related to a problem? Please describe.**
yes.

**Describe the solution you'd like** 💡
i will create a bash script to make url opening easier

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered.

**Additional context**
Add any other context or screenshots about the feature request here.
